profile/README.md

```diff
@@ -1,3 +1,9 @@
-# Welcome
+[![Source Allies](https://raw.githubusercontent.com/sourceallies/.github/main/profile/sa-header.png)](https://sourceallies.com/)
 
-[![Source Allies](https://raw.githubusercontent.com/sourceallies/.github/main/profile/sa-logo.svg)](https://sourceallies.com/)
+Source Allies is a technology services company, powered by people in slippers!
```

Comment:
"technology services company" is rather vague. Do we have a better definition? 

I wonder if we want to add in anything surrounding our goal of being the preferred place of employment for collaborative technologists seeking mastery.

Thank you for working on this!

Reply:
I like that! Lemme swap that up and add it. I just copy/pasted from our front page but I can wordsmith a bit 😄